
(25, 'Inappropriate ioctl for device')

Unable to access any of "/cluster/home/julievt/.cache/pypoetry/virtualenvs/master-project-118ajfdt-py3.12/lib/python3.12/site-packages/master_project-0.1.0.dist-info"
[nltk_data] Downloading package punkt to
[nltk_data]     /cluster/home/julievt/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Traceback (most recent call last):
  File "/cluster/home/julievt/master-project/./scripts/active-learning/ner.py", line 11, in <module>
    os.mkdir(f'./data/helsearkiv/batch/ner/{BATCH}')
FileExistsError: [Errno 17] File exists: './data/helsearkiv/batch/ner/5'
